.storybook/preview.js

```diff
@@ -44,18 +44,17 @@ export const parameters = {
             [
               '*',
               // Within a set of stories, set the order to the following
-              ['*', 'Playground', /Playground$/, 'Features', 'Examples'],
+              ['*', 'Playground', /Playground$/, 'Features', 'Examples', 'DevOnly'],
             ],
           ],
         ],
+        'Drafts',
```

Comment:
I can't make the special order of 'Components' apply to the 'Drafts' as well. Any ideas how can I do that? 👀 cc @langermank

Reply:
What if you copy the exact thing from `Components` into `Drafts`?

```
[
            [
              '*',
              // Within a set of stories, set the order to the following
              ['*', 'Playground', /Playground$/, 'Features', 'Examples', 'DevOnly'],
            ],
          ],
```